projects/m1/031-coin-flip-simulation/solutions/bisicchiamatteo/031-coin-flip-simulation.py

```diff
@@ -0,0 +1,27 @@
+import random
+total_f = 0
+
+for flip_cont in range(11):
+    i_tail = 0
```

Comment:
Come nei precedenti esercizi, non è chiaro il significato delle variabili.

In ambiente lavorativo, su progetti grossi su cui magari metti mano anche dopo mesi, è importante dare nomi auto-esplicativi, questo rende il codice più leggibile e manutenibile.